code/controllers/Processes/mob.dm

```diff
@@ -17,4 +17,4 @@
 /datum/controller/process/mob/doWork()
 	if(updateQueueInstance)
 		updateQueueInstance.init(mob_list, "Life")
```

Comment:
updateQueueInstances are removed from the latest process scheduler, change it to a normal loop with proper schecks.